pkg/localizer/localizer.go

```diff
@@ -0,0 +1,37 @@
+// Package localizer is meant to contain useful helper functions, variables, and
+// constants in order to better programatically interact with localizer.
+package localizer
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	apiv1 "github.com/getoutreach/localizer/api/v1"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+// Socket is the communication endpoint that the localizer server is listening
+// on.
+const Socket = "/var/run/localizer.sock"
+
+// IsRunning checks to see if the localizer socket exists.
+func IsRunning() bool {
```

Comment:
thinking about renaming this `SocketExists` and then creating a better `IsRunning` function that quickly connects to the localizer server, pings it, and returns. what do you think @jaredallard ?

Reply:
That's fine, but this is also OK to me.

Reply:
eh i'll leave it then